src/domain/behavior/behavior-table-tile.tsx

```diff
@@ -0,0 +1,190 @@
+];
+
+interface BehaviorTileProps {
+  text: GedragText;
+  behavior: BehaviorValue;
+}
+
+interface BehaviorFormatted {
+  id: BehaviorIdentifier;
+  description: string;
+  percentage: number | null;
+  trend: BehaviorTrendType;
+}
+
+const MobileScroll = styled.div(
+  css({
+    overflow: 'auto',
+  })
+);
+
+const Table = styled.table(
+  css({
+    width: '100%',
+  })
+);
+
+const HeaderCell = styled.th(
+  css({
+    px: 3,
+    py: 2,
+    textAlign: 'left',
+  })
+);
+
+const Cell = styled.td(
+  css({
+    borderBottom: '1px solid',
+    borderBottomColor: 'lightGrey',
+    px: 3,
+    py: 2,
+  })
+);
+
+const Footnote = styled.p(
+  css({
+    color: 'gray',
+  })
+);
+
+/* Format raw list of behaviors into list for compliance or for support */
+function formatBehaviorType(
+  behavior: BehaviorValue,
+  type: 'compliance' | 'support'
+): BehaviorFormatted[] {
+  return behaviorIdentifiers.map((identifier) => {
+    const percentage = behavior[
+      `${identifier}_${type}` as keyof BehaviorValue
+    ] as number | null;
+    const trend = behavior[
+      `${identifier}_${type}_trend` as keyof BehaviorValue
+    ] as BehaviorFormatted['trend'] | null;
+
+    return {
+      id: identifier,
+      description: siteText.gedrag_onderwerpen[identifier],
+      percentage,
+      trend,
+    };
+  });
+}
+
+/* Sort lists of formatted compliance and support behaviours */
+function sortBehavior(
+  compliance: BehaviorFormatted[],
+  support: BehaviorFormatted[]
+): {
+  sortedCompliance: BehaviorFormatted[];
+  sortedSupport: BehaviorFormatted[];
+} {
+  /* Sort compliance on percentage, decreasing; percentage can be null */
+  const sortedCompliance = compliance.sort(
+    (a, b) => (b.percentage ?? 0) - (a.percentage ?? 0)
+  );
+
+  /* Sort support based on the order of compliance */
+  const sortedSupport = support.sort((a, b) => {
+    return (
+      sortedCompliance.findIndex((x) => x.id === a.id) -
+      sortedCompliance.findIndex((x) => x.id === b.id)
+    );
+  });
+
+  return { sortedCompliance, sortedSupport };
+}
+
+function formatAndSortBehavior(
+  behavior: BehaviorValue
+): {
+  sortedCompliance: BehaviorFormatted[];
+  sortedSupport: BehaviorFormatted[];
+} {
+  const compliance = formatBehaviorType(behavior, 'compliance');
+  const support = formatBehaviorType(behavior, 'support');
+
+  return sortBehavior(compliance, support);
+}
+
+export function BehaviorTableTile({ text, behavior }: BehaviorTileProps) {
+  const { sortedCompliance, sortedSupport } = formatAndSortBehavior(behavior);
+  const [behaviorType, onChangeControls] = useState<'compliance' | 'support'>(
+    'compliance'
+  );
+
+  return (
+    <Tile>
+      <h3>{text.basisregels.title}</h3>
+      <Box display="flex" justifyContent="start">
+        <BehaviorTypeControls
+          onChange={onChangeControls}
+        ></BehaviorTypeControls>
+      </Box>
+
+      <p>{text.basisregels.intro[behaviorType]}</p>
+      <MobileScroll>
+        <Table>
```

Comment:
another nitpick thing: since these components almost have no styling and they're used on a single place, it would imho be fine to write the css inline.

eg:

```suggestion
      <div css={css({ overflow: 'auto' })}>
        <table css={css({ width: '100%' })}>
```

It's not really necessary to rewrite it, but it might improve development speed when you're writing tiny bits of css like this.